steward/src/cork/proposals.rs

```diff
@@ -251,6 +251,33 @@ async fn poll_approved_cork_proposals(state: &mut ProposalThreadState) -> Result
                     }
                 }
             }
+            Call::CellarV2(data) => {
+                if data.function.is_none() {
+                    warn!(
+                        "scheduled cork proposal {} call data contains no function data and will be ignored: {:?}",
+                        proposal.proposal_id, data,
+                    );
+                    state.increment_proposal_id();
+                    continue;
+                }
+                let function = data.function.unwrap();
+                match cellar_v2::get_encoded_governance_call(
+                    function,
+                    &cellar_id,
+                    proposal.proposal_id,
+                ) {
+                    Ok(d) => d,
+                    // this is likely a bug in steward
+                    Err(err) => {
+                        error!(
+                            "failed to get encoded governance call data for proposal {}: {}",
+                            proposal.proposal_id, err
+                        );
+                        state.increment_proposal_id();
```

Comment:
Did we ever account for the fact that there may be missing proposal ID values in a sequential processing of them?